clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -3409,3 +3409,30 @@ StringRef tools::parseMRecipOption(clang::DiagnosticsEngine &Diags,
 
   return Out;
 }
+
+std::string tools::ComplexRangeKindToStr(LangOptions::ComplexRangeKind Range) {
```

Comment:
The functions in `CommonArgs.cpp` use a lower-case letter as the first letter in the function name, unlike clang which often uses upper case. 

```suggestion
std::string tools::complexRangeKindToStr(LangOptions::ComplexRangeKind Range) {
```

Reply:
Thank you for the review. I have corrected the function names.